bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/DropDownImpl.java

```diff
@@ -43,6 +46,10 @@ public class DropDownImpl extends AbstractOptionsFieldImpl implements DropDown {
     @Default(booleanValues = false)
     protected boolean multiSelect;
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL, name = ReservedProperties.FD_ENABLE_SUGGESTIONS)
+    @Default(booleanValues = false)
+    protected boolean enableSuggestions;
```

Comment:
The default value should not be part of JSON, since it increases the JSON payload. You can refer the existing visible/enable implementation on how we did this

Reply:
okay, will fix it.